apis/Google.Cloud.Firestore/Google.Cloud.Firestore/Query.cs

```diff
@@ -509,6 +511,38 @@ object ConvertValueToDocumentReferencesForInQuery()
             }
         }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="filter"></param>
+        /// <returns></returns>
+        public Query Where(Filter filter)
+        {
+            ValidateFilterForSentinels(filter);
+            var newFilter = (_filter == null) ? new Filter.CompositeFilter(new List<Filter>(), CompositeFilter.Types.Operator.And)

```

Comment:
Presumably you could just use `new List<Filter> { filter }` rather than calling `WithAdditionalFilter`?

Or you could use:

```csharp
var newFilter = (_filter ?? new Filter.CompositeFilter(new List<Filter>(), CompositeFilter.Types.Operator.And))
    .WithAdditionalFilter(filter);
```

(We might want a simple `Filter.EmptyAnd` or similar factory method, if we use it a lot...)